migrator/README.md

```diff
@@ -0,0 +1,63 @@
+# UnifiedPush Server - Migration guide
+
+For the database migration you may need to run a simple shell script
+
+# 1.0.x users
+
+For the migration to our 1.0.3 release we are only supporting the 1.0.x series of the UnifiedPush Server:
+
+* 1.0.0
+* 1.0.1
+* 1.0.2
+
+In terms of supported databases, we are supporting MySQL 5.5 and PostgreSQL 9. 
```

Comment:
well, not sure... on explicit MariaDB mentioning here... Opebshift still
labels its DB as MySQL

On Wed, Mar 11, 2015 at 10:54 AM, Sebastien Blanc <notifications@github.com>
wrote:

> In migrator/README.md
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/510#discussion_r26199568>
> :
>
> > @@ -0,0 +1,63 @@
> > +# UnifiedPush Server - Migration guide
> > +
> > +For the database migration you may need to run a simple shell script
> > +
> > +# 1.0.x users
> > +
> > +For the migration to our 1.0.3 release we are only supporting the 1.0.x series of the UnifiedPush Server:
> > +
> > +* 1.0.0
> > +* 1.0.1
> > +* 1.0.2
> > +
> > +In terms of supported databases, we are supporting MySQL 5.5 and PostgreSQL 9.
>
> Should we mention MariaDB is also part of "MySQL" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/510/files#r26199568>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf